### ISSUE TITLE
Empty results

### ISSUE THREAD

**Issue by dawiepoolman** [thiago-js/scraping-google-maps#1]
@thiago-js
I have compiled the code but when it runs nothing happens.  After s few seconds he code completes but no file gets generated under /scrap/results
Any debugging clues?

**Comment by yynfz** [comment]
I have the same issue

> @thiago-js
> I have compiled the code but when it runs nothing happens. After s few seconds he code completes but no file gets generated under /scrap/results
> Any debugging clues?

**Comment by fugaswolf** [comment]
does this thing work?

**Comment by danishfareed** [comment]
It did work for me for a few days. But now the same issue as others. Trying to figure out what exactly the problem is.

**Comment by Quest1mc** [comment]
seems like a really cool program it would really help if this was running . I'll do some digging around ...not that I'm a pro or anything but it would be a really nice tool to have

**Comment by dawiepoolman** [comment]
this project seems o be dead.
I found this one more promising https://github.com/serp-spider

**Comment by Quest1mc** [comment]
I actually got it to work using the code you added plus I noticed that it was not creating the file. so I went in and added the folder and files .
next thing I noticed is the tolocalestrig method used actually is a problem since it includes slashes in the file name and slashes are not allowed in file names so i took it out  you can see where I put it in comments 
`fs.appendFileSync(
        // "./scrap/results/scrap_" + new Date().toLocaleDateString() + ".txt",
        "./scrap/results/scrap_" + new Date() + ".txt",
`

**Comment by danishfareed** [comment]
@Quest1mc Yes I know this error the slashes are not allowed in the file name.

Note: To execute this script first, create a file (sracp.txt) in the "result" folder. Because we are only appending data to a file in the code, we are not creating a new one.

Then make this change in the code:

fs.appendFileSync(// "./scrap/results/scrap_" + new Date().toLocaleDateString() + ".txt"

"./scrap/results/scrap.txt",

change it with the above code. You'll append all data in a single file. 
To create a new file every time you can do this:

"let randomnum = Math.floor((Math.random() * 11000) + 1);" add this code some where in the starting of the code, it shouldn't be in that loop. then use "x" as file name like this:

"./scrap/results/"+randomnum+".txt",

**Comment by thiago-js** [comment]
> this project seems o be dead.
> I found this one more promising https://github.com/serp-spider

**Comment by thiago-js** [comment]
257/5000
Excuse me friends, I promise that I will solve all the problems and leave the repository in a position to obtain the contacts.

Sorry for the immense delay in the return, but today I'm going to check what is happening and already send a return to you all right.

**Comment by thiago-js** [comment]
> @thiago-js
> I have compiled the code but when it runs nothing happens. After s few seconds he code completes but no file gets generated under /scrap/results
> Any debugging clues?

**Comment by thiago-js** [comment]
> @thiago-js
> I have compiled the code but when it runs nothing happens. After s few seconds he code completes but no file gets generated under /scrap/results
> Any debugging clues?

Sorry about that man, I promise resolved the problem today all right.
